src/components/tabs/__internal__/tab-title/tab-title.style.ts

```diff
@@ -55,6 +55,12 @@ const StyledTitleContent = styled.span<StyledTitleContentProps>`
       justify-content: ${align === "right" ? "flex-end" : "flex-start"};
     `}
 
+    ${position === "top" &&
```

Comment:
Validation styling is missing when left positioned

![image](https://user-images.githubusercontent.com/44157880/228322701-3d818c16-a6a0-4e0d-9068-1e68395c1b7f.png)